model_bmi/build_thrift.xml

```diff
@@ -91,11 +91,12 @@
         </antcall>
     </target>
 
-    <!-- This only works if thrift compiler is on the PATH environment variable and output directory already exists -->
+    <!-- This only works if output directory already exists -->
     <target name="thrift-execute">
 		<echo>Generating ${output_language} code in ${output_dir} using thrift code from ${input_file}</echo>
-        <exec executable="thrift" failonerror="${failonerror}">
-            <arg line="-out ${output_dir}"/>
+      <exec executable="${resourcesdir}\thrift\thrift-0.11.0.exe" failonerror="${failonerror}">
```

Comment:
This only works under Windows. However, it does solve the versioning problem with thrift. @nilsvanvelzen what do you think?